File: post-service/src/main/java/com/muates/postservice/model/dto/response/PostCommentResponse.java
package com.muates.postservice.model.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PostCommentResponse {

    private Long id;
    private Long userId;
    private String content;
    private Date createdDate;
    private Date updatedDate;
    private List<PostCommentLikeResponse> likes;
}

File: front-end/src/app/core/models/post/response/post-comment-response.ts
import { PostCommentLikeResponse } from "./post-comment-like-response";

export class PostCommentResponse {
    id!: number;
    userId!: number;
    content!: string;
    createdDate!: Date;
    updatedDate!: Date;
    likes!: PostCommentLikeResponse[];
    username!: string;
    profilePicture!: string;
}
